### PR TITLE
Redirected the url to the Jfrog sign up page.

### PR DESCRIPTION
**Pull request by eyalk007**
This branch switches the creation of a new env for free, for a creation as part of a free trial.
Today the feature doesn't work as JFrog does not support free envs for an unlimited time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,6 @@ Using the JFrog Extension doesn't require a paid JFrog subscription.
 Follow these directions to proceed, based on your use case:
 
 - If you already have a JFrog environment which includes Xray, all you need to do is set the connection details through the UI.
-- If you don't have a JFrog environment, JFrog supports setting up an environment for free. The extension will then connect to this environment automatically. Please note that this new JFrog environment will be available for you as long as you need it.
+- If you don't have a JFrog environment, JFrog supports setting up an environment with a free trial. The extension will then connect to this environment automatically. Please note, if no payment method is added, that this new JFrog environment will not be available after the trial period ends.
```
Comment (hadarshjfrog) [inline README.md:44]: @asafcjfrog asked to shorten to:
```suggestion
-  If you don't have a JFrog environment, you can try it out with JFrog's free trial.
```

**Comment by hadarshjfrog** [comment]
Please add a screenshot(s) of how the new page will look like - we want to keep it well documented

**Comment by eyalk007** [comment]
credentials page -
 image

**Comment by eyalk007** [comment]
details page - 
image

**Comment by eyalk007** [comment]
free trial env link page - 
image

**Review by hadarshjfrog — APPROVED** [review]

**Pull request merged**